### ISSUE TITLE
Update Contribution Guidelines

### ISSUE THREAD

**Issue by smaranjitghose** [smaranjitghose/awesome-portfolio-websites#756]
# Requirements:

- Exclusively work with [CONTRIBUTING.md](https://github.com/smaranjitghose/awesome-portfolio-websites/blob/master/CONTRIBUTING.md)
- All the images/clips used should be inside ``./readme_assets/contributing/``
- Divide the Guide into two sections

```
# A. Setting up the Project

1. Fork
2. Clone
3. Move inside the folder
4. Check Remotes
5. Add Remote to Upstream to pull changes from the original repo
6. Open VS Code
7. Run Live Server [Install it from Extensions Marketplace if not done already]

# B. Contributing to the Project

1. Pull the new changes
2. Make a feature branch
3. Do the changes
4.  Stage them
5.  Commit
6. Push it to the forked remote
7. Compare it with the desired branch
8. Make PR

# C. Instructions for PR
- Title should be brief and accurate
- Refer the issue
- Description is must. Should be in bullet points, short phrases
- Clips/snapshots before and after changes

# D. General Guidelines:

( Check my pinned message on slack for the content)

```

- For the steps like forking, cloning, raising a PR add gifs (i.e. Record Short clips using OBS Studio and convert them into GIF)
- Check my pinned message on slack for detailed content
- The above text inside the block is just a sample. __DO NOT DIRECTLY COPY PASTE. It should be elaborated___
- Remove the Topmost Section, Broken Gif and Illustration with a centre aligned image with a text over it "Contribution Guidelines)

**Comment by ThejasKiranPS** [comment]
May I add the gifs?

**Comment by ishitakapoor26** [comment]
pls assign me this issue.

**Comment by smaranjitghose** [comment]
It's been 2 days @ishitakapoor26. This is a really simple issue.  You have 1 more day for this

**Comment by ishitakapoor26** [comment]
> It's been 2 days @ishitakapoor26. This is a really simple issue. You have 1 more day for this

okay

**Comment by Amrutha1101** [comment]
Can I work on this issue under LGM-SoC ?

**Comment by deeqakkk** [comment]
@Amrutha1101  Yes, you can work on it.

**Comment by Amrutha1101** [comment]
Can you re share the pinned message here for detailed content as slack isn't used now?

**Comment by deeqakkk** [comment]
@Amrutha1101 https://app.slack.com/client/T022A4RL16V/C0246EFQG4T
Pinned message can be found here.

**Comment by Amrutha1101** [comment]
Thanks

**Comment by Amrutha1101** [comment]
I guess the message isn't visible though it's pinned

**Comment by smaranjitghose** [comment]
> I guess the message isn't visible though it's pinned

Leave that. I'll take care of it. Meanwhile, you can complete the rest @Amrutha1101

**Comment by smaranjitghose** [comment]
Common Guidelines:

A) Pull Request:

- All Pull Requests should have a single commit. In case of multiple commits, please perform a rebase and make a squash commit before giving the Pull Request
- The Pull Request title should be appropriate and relatable to the issue under which it was assigned to the participant
- Pull Requests without a description would often not be reviewed. The work done in the PR should be elaborated in phrases using bullet points. The corresponding issue should be linked in the following manner fixes #issue_number.  For Example: fixes #646 . This is usually the first line for the PR description before the synopsis of the work done. Lastly, relevant screenshots/short clips of the changes due to the code fix/update should be attached.

Sample PR Description

Fixes #640

# Work Done:

- Fixed Responsiveness bug in the MOOC Section of the Education Page
- Made the Favicon more optimized
- ...
- ...
- ...

# Relevant Screenshots/Gifs

- After the PR is raised, refrain from putting up messages like "Please review this PR", etc either on the corresponding issue, this slack workspace or the PR itself. All the project maintainers are full time research scientists/software developers and would review the PR within 24-72 hours of it being raised as per their convenience.
- Not every PR would be merged directly. In majority of the scenarios, we would be offering some additional suggestions which are to be incorporated in the same PR with a squash commit.
- Look out for possible merge conflicts
- The code added should be well documented (Add Comments)
- Indentation/Linting of the code should not be tampered with
- Make sensible variable names. For example, card1 is quite ambiguous but education_card implies what that variable is declared for

B) Issues

- Whenever a participant raises an issue, by default it would be assigned to that participant due to obvious reasons. Please do not spam the issue thread by blindly commenting "I want to work on this". "Assign me this issue" etc. None of these requests would be entertained
- You are welcome to comment on the issues raised by the project maintainers i.e. me and @Anush Bhatia[Project Admin] for claiming them. 
- Please be patient for the issues to be evaluated and/or assigned. All the project maintainers are full time research scientists/software developers and would review the issue comments within 24-36 hours of them being posted as per their convenience. Refrain from putting up messages like "Please review this PR", etc either on the issue or this slack channel.
- An issue would be assigned to a participant for a minimum of 1 day and a maximum of 3 days. If the tenure of a particular participant is not over please do not spam the issue thread by writing, " I am interested" etc. You are welcome to do so if the participant has not met the requirements with around 12 hours remaining for their deadline.
- Refrain from putting up messages like "Thank You" anywhere in the issue or PR threads since we would get unnecessary notifications for those.
- Feel free to suggest your opinions on the already raised issues
- All participants are welcomed to suggest new features/bug fixes/improvements and raise new issues for those. However, the assignment would be done once approved by either of the project maintainers or the mentors.
- Make use of this slack channel for doubt clearing and discussion of concepts related to the work.

C) Slack

- Please put the entire text in a single message. Use Shift+Enter to move to the next line and Enter to send the message.
- For replying to a particular message, please do so in threads
- Avoid giving your intro for asking doubts. Keep the discussions concise.
- Unless there is a personal issue, no DMs would be entertained by the project maintainers. All project related conversations should be kept exclusively to the respective slack channels.

**Comment by smaranjitghose** [comment]
@Amrutha1101 I have pasted that message here
